libs/domain/search/facet-rating/facet-rating.def.ts

```diff
@@ -0,0 +1,9 @@
+import { componentDef } from '@spryker-oryx/utilities';
+
+export const searchRatingFacetComponent = componentDef({
+  name: 'oryx-search-rating-facet',
```

Comment:
Why the name of the component is `oryx-search-rating-facet` but the folder name is `facet-rating`? Please make it consistent.

Reply:
I took the example of the naming from `facet-color`, but I will rename those components to `oryx-search-facet-rating` and `oryx-search-facet-color`

Reply:
We can't rename existing one. What we can do - we can change folder name as long as it's not a public API.